config/locales/en.yml

```diff
@@ -73,6 +73,9 @@ en:
       tagline: Turns out websites are expensive and ads suck
       content_html: Support Workshop.codes for just <span class="text-white">$3</span>
       button: Support Workshop.codes
+    overwatch_2:
+      title_html: <span class="text-overwatch-2">Overwatch 2</span> Beta compatible&nbsp;codes
```

Comment:
```suggestion
      title_html: <span class="text-overwatch-2">Overwatch 2</span> compatible&nbsp;codes
```

Reply:
I kept Beta in here for SEO. I'm assuming people might search google for "overwatch 2 beta workshop" or something along those lines.

Reply:
Ah ok, then that's fine